website_sale_force_qty/static/src/js/website_sale_force_qty_tour.js

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2021 Tecnativa - João Marques
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+ odoo.define("website_sale_force_qty.tour", function (require) {
+    "use strict";
+
+    var tour = require("web_tour.tour");
+    var base = require("web_editor.base");
+
+    var steps = [
+        {
+            trigger: "a:contains('Test product with fixed quantity')",
+        },
+        {
+            extra_trigger: [
+                "div.js_main_product",
+                // The quantity input box is disable and is set to 2
+                ":has(input[name=add_qty][value='2']:disabled)",
+                // There is a badge indicating the product as a fixed quantity
+                ":has(.text-info:contains('This product has a fixed quantity'))",
```

Comment:
Maybe this text should be customizable to allow further explanations

Reply:
Mmm yes, it could be, adding it to the roadmap :+1: